### PR TITLE
 `src/platforms/` (rewards) refactor

### PR DESCRIPTION
**Pull request by dominik-stumpf**

### CODE REVIEW

**Comment by dominik-stumpf** [comment]
For now separated files such as `constants.ts` is duplicated in `rewards.ts`. This will be addressed in the following PR.

File: src/platforms/rewards.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import pointsReward from "platforms/Points"
+import polygonIdReward from "platforms/PolygonID"
+import uniqueTextReward from "platforms/UniqueText"
+import textReward from "platforms/SecretText"
+import contractCallReward from "platforms/ContractCall"
+import googleReward from "platforms/Google"
+import discordReward from "platforms/Discord"
+import githubReward from "platforms/Github"
+import poapReward from "platforms/Poap"
+// import { Reward } from "./types"
```
Comment (BrickheadJohnny) [inline src/platforms/rewards.ts:16]: I think you can remove this commented line

File: src/platforms/rewards.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+export type CardPropsHook = (guildPlatform: GuildPlatformWithOptionalId) => {
+  type: PlatformName
+  name: string
+  image?: string | JSX.Element
+  info?: string | JSX.Element
+  link?: string
+  shouldHide?: boolean
+}
+
+export const modalSizeForPlatform = (platform: PlatformName) => {
```
Comment (BrickheadJohnny) [inline src/platforms/rewards.ts:126]: Let's add a comment about the duplication inside the code (instead of just the PR) 👌

**Review by BrickheadJohnny — APPROVED** [review]
Left 2 comments, feel free to merge the PR once you fixed those ;)

**Pull request merged**